biome.json

```diff
@@ -0,0 +1,111 @@
+{
+  "$schema": "./node_modules/@biomejs/biome/configuration_schema.json",
+  "files": {
+    "includes": [
+      "**",
+      "!**/node_modules",
+      "!**/dist",
+      "!**/package-lock.json",
+      "!**/yarn.lock",
+      "!**/pnpm-lock.yaml",
+      "!**/bun.lockb",
+      "!**/output",
+      "!**/coverage",
+      "!**/temp",
+      "!**/.temp",
+      "!**/tmp",
+      "!**/.tmp",
+      "!**/.history",
+      "!**/.vitepress/cache",
+      "!**/.nuxt",
+      "!**/.next",
+      "!**/.vercel",
+      "!**/.changeset",
+      "!**/.idea",
+      "!**/.cache",
+      "!**/.output",
+      "!**/.vite-inspect",
+      "!**/.yarn",
+      "!**/CHANGELOG*.md",
+      "!**/*.min.*",
+      "!**/LICENSE*",
+      "!**/__snapshots__",
+      "!**/components.d.ts",
+      "!**/mockServiceWorker.js"
+    ]
+  },
+  "vcs": {
+    "clientKind": "git",
+    "enabled": true,
+    "useIgnoreFile": true,
+    "defaultBranch": "master"
+  },
+  "formatter": {
+    "enabled": true,
+    "formatWithErrors": true,
+    "indentStyle": "space",
+    "indentWidth": 2,
+    "lineEnding": "lf",
+    "lineWidth": 80,
```

Comment:
oh, can we bump this to 120? 80 chars dates back to old 600x400 CRT days and for (almost?) everyone it's way narrower  than their editor views. I default (where I have the chance, either in fmt options or by having lines drawn in my editor) to either 100 or 120.

Reply:
will change to 120